public/js/main.js

```diff
@@ -21,4 +23,24 @@ $(function() {
 			headerScroll();
 	}
     $('[data-toggle="tooltip"]').tooltip();
+	$('.collapse')
+		.collapse('hide')
+		.on('show.bs.collapse', () => {
```

Comment:
I just tried deploying it. We don't support ES6 constructs yet. I'll need to replace grunt with something else before we can support ES6.